src/modules/RA/DataSource/index.js

```diff
@@ -12,15 +12,29 @@ export const GEOJSON = 'GeoJSONSource';
 export const SQL = 'PostGISSource';
 export const WMTS = 'WMTSSource';
 export const SHP = 'ShapefileSource';
+export const CSV = 'CSVSource';
+export const UTF8 = 'UTF-8';
+export const LATIN1 = 'Latin-1';
+export const EPSG4326 = 'EPSG:4326 - WGS 84';
```

Comment:
Il faudrait rendre les constantes qui seront utiilisée par le back plus facile à écrire. Par exemple :
```suggestion
export const EPSG4326 = 'EPSG_4326';
```

Reply:
ok